semantic-python/src/Language/Python/ScopeGraph.hs

```diff
@@ -87,7 +87,7 @@ scopeGraphModule = getAp . scopeGraph
 instance ToScopeGraph Py.AssertStatement where scopeGraph = onChildren
 
 instance ToScopeGraph Py.Assignment where
-  scopeGraph (Py.Assignment _ (SingleIdentifier t) _val _typ) = complete <* declare @Name t DeclProperties
+  scopeGraph (Py.Assignment _ (SingleIdentifier t) _val _typ) = complete <* declare @Name (formatName t) DeclProperties
```

Comment:
I’m not sure I understand why this takes `Text` if we’re instantiating the type var to `Name`?

Reply:
Yeah, that’s a wart. I can fix it here but left it just in case @joshvera was changing it too.